articles/firewall-manager/migrate-to-policy.md

```diff
@@ -110,7 +114,18 @@ If ($azfw.NetworkRuleCollections.Count -gt 0) {
 			Write-Host "creating " $rc.Rules.Count " network rules for collection "  $rc.Name
 			$firewallPolicyNetRules = @()
 			ForEach ($rule in $rc.Rules) {
+				If(($rule.SourceAddresses -and $rule.DestinationAddresses)){
 				$firewallPolicyNetRule = New-AzFirewallPolicyNetworkRule -Name $rule.Name -SourceAddress $rule.SourceAddresses -DestinationAddress $rule.DestinationAddresses -DestinationPort $rule.DestinationPorts -Protocol $rule.Protocols
+				}
+			    elseif(($rule.SourceIpGroups -and $rule.DestinationAddresses)){
```

Comment:
can we please restructure this a bit? Let have it this way:
If($rule.SourceAddresses){
      If($rule.DestinationAddresses)
      {
             <code here>
       }
      elseif($rule.DestinationIpGroups)
      {
             <code here>
        }
}else if(($rule.SourceIpGroups){
      If($rule.DestinationAddresses)
          {
             <code here>
         }
      elseif($rule.DestinationIpGroups)
         {
            <code here>
        .}
}

Reply:
Agreed. Fixed